src/Controller/LogVendorController.php

```diff
@@ -65,11 +65,24 @@ protected function prepareAjaxData($data)
                 "label" => $this->getStatusString($logRow['status']),
                 "button" => $this->getStatusMessage($logRow)
             );
+
+            $data[$key]['enabled'] = array(
+                "enabled" => $logRow['enabled'],
+                "label" => $this->getEnableLabel($logRow['enabled'])
+            );
         }
 
         return $data;
     }
 
+    private function getEnableLabel($enable){
+        if($enable){
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Put one space between the closing parenthesis and the opening curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1808)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Put one space between this "if" keyword and the opening parenthesis. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1808)